src/App.jsx

```diff
@@ -1,35 +1,37 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState } from "react";
+import "./App.css";
+import { Navbar } from "./components/Navbar";
+import axios from "axios";
+import { Articles } from "./components/articles";
+import { Article } from "./components/Article";
+import { Route, Routes } from "react-router-dom";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
```

Comment:
As our app has a useEffect making an api call, loading state should start off as true